community.md

```diff
@@ -20,14 +20,35 @@ page](gsoc.html).
 All mlpack development is done on [GitHub](https://github.com/mlpack/mlpack).
 In addition to that, we use:
 
- * the [mlpack mailing list](http://lists.mlpack.org/mailman/listinfo/mlpack)
-   for **email discussion and announcements**,
- * the [#mlpack](http://webchat.freenode.net/?channels=mlpack) IRC channel on
-   freenode for **real-time chat and discussion** ([logs](http://www.mlpack.org/irc/)),
- * the [mlpack-git mailing list](http://lists.mlpack.org/mailman/listinfo/mlpack-git),
-   which sends email for **all mlpack-related activity**.
- * the [mlpack blog](http://www.mlpack.org/blog/) for **mlpack development
-   updates and blog posts**
+<ul class="flex-container">
+  <li class="flex-item">
+      <div class="card">
+	    <i class="far fa-envelope fa-lg card-icon"></i>
+	    <p><a href="http://lists.mlpack.org/mailman/listinfo/mlpack">mlpack mailing list</a></p><p>for email discussion and announcements</p>
+	  </div>
+  </li>
+  
+  <li class="flex-item">
+      <div class="card">
+	    <i class="far fa-comments fa-lg card-icon"></i>
+	    <p><a href="http://webchat.freenode.net/?channels=mlpack">#mlpack IRC channel</a></p><p>on freenode for real-time chat and discussion <a href="http://www.mlpack.org/irc/">(logs)</a></p>
+	  </div>
+  </li>
+  
+  <li class="flex-item">
+      <div class="card">
+	    <i class="fab fa-git-square fa-lg card-icon"></i>
+	    <p><a href="http://lists.mlpack.org/mailman/listinfo/mlpack-git">mlpack-git mailing list</a></p><p>which sends email for all mlpack-related activity</p>
+	  </div>
+  </li>
+  
+  <li class="flex-item">
+      <div class="card">
+	    <i class="far fa-newspaper fa-lg card-icon"></i>
+	    <p><a href="http://www.mlpack.org/blog/">Blog</a></p><p>for mlpack development updates and blog posts</p>
+	  </div>
+  </li>
+</ul>
```

Comment:
Agree, and done. BTW, I couldn't find the chat services changes in the master community.md file, did you forget to push? I ended up copying and pasting from the live website anyways. All done.

Reply:
Yeah, like I said in another comment, I haven't pushed because I didn't do my homework yet to figure out how to push back to this repo without including the huge LFS objects. :(  I'll get it sorted...